_extras/checkout.md

```diff
@@ -45,45 +49,33 @@ with our collaborative curriculum development process.
 
 Trainees must make a contribution to a lesson's content, exercises, or instructor's guide by doing one of the following:
 
-1.  Submitting a change request to fix an existing issue.
+1.  Submitting a change to fix an existing issue.
 2.  Proof-reading a lesson and adding a new issue describing something to be improved.
-3.  Providing substantive feedback on an existing issue or PR.
+3.  Providing substantive feedback on an existing issue or pull request.
+
+Your contribution must be for a lesson in the curriculum of the Carpentry (Software or Data) for which you are 
+qualifying. Instructors seeking to qualify for both Carpentries, must submit a contribution to both a Data Carpentry
+and a Software Carpentry lesson.
+
+Data Carpentry instructors may submit their contributions by [email](mailto:{{ site.email }}) or through GitHub.
+Software Carpentry instructors must submit their contributions through GitHub.
 
 Trainees must do their work on one of the lessons for which we are currently seeking improvements.
-We encourage trainees to submit their changes through GitHub,
-but will accept submissions for #1 and #2 via email from those who are more comfortable working out of public view.
```

Comment:
I would like to keep this sentence, because GitHub submissions are a lot easier for us to manage.